data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java

```diff
@@ -48,6 +48,11 @@
      */
     protected double voltageLevel;
 
+    /**
+     * Flow reliability margin of the Cnec associated to this threshold, in MEGAWATT
+     */
+    protected double frmInMW;
```

Comment:
It seems quite tricky with the voltageLevel which is in this class. But isn't there a way to avoid the duplication of the frm (which is here and in the SimpleCnec) ?

The clean way would be to put the voltageLevel in the Cnec, but this is quite a huge refactoring. And I don't think we want to do this now.

Anyway, if we keep this parameter I think we could add a in the JSON import/export a case with a Cnec which as FRM != 0.